README.md

```
@@ -252,20 +252,30 @@ This method is recommended for developers who want to:
    ```
 
-5. **Start the Development Server**:
+5. **(OPTIONAL)** Switch to the Main Branch if you want to use pre-release/testbranch:
    ```bash
+   git checkout main
+   pnpm install
    pnpm run dev
    ```
+  Hint: Be aware that this can have beta-features and more likely got bugs than the stable release
+
+>**Open the WebUI to test (Default: http://localhost:5173)**
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)